web-client/src/presenter/actions/shouldSaveToConsolidatedGroupAction.ts

```diff
@@ -13,14 +13,15 @@ export const shouldSaveToConsolidatedGroupAction = ({
   get,
   path,
 }) => {
-  const { UNSERVABLE_EVENT_CODES } = applicationContext.getConstants();
+  const { OPINION_PAMPHLET_EVENT_CODE, UNSERVABLE_EVENT_CODES } =
+    applicationContext.getConstants();
 
   const caseDetail = get(state.caseDetail);
   const { eventCode } = get(state.form);
 
   const isUnservable = UNSERVABLE_EVENT_CODES.includes(eventCode);
   const isLeadCase = applicationContext.getUtilities().isLeadCase(caseDetail);
-  const isNotTaxCourtPamphlet = eventCode !== 'TCRP';
+  const isNotTaxCourtPamphlet = eventCode !== OPINION_PAMPHLET_EVENT_CODE;
 
   if (isLeadCase && isUnservable && isNotTaxCourtPamphlet) {
```

Comment:
🔧 maybe instead TCRP could be included as an UNSERVABLE_EVENT_CODE?

Reply:
Good idea, looking at the uses of UNSERVABLE_EVENT_CODE - I see it being used to hide/show things in the system that don't always apply to TCRPs, so I think it makes more sense to leave it out of that list if that sounds good to you